sm/engine/acq_geometry_factory.py

```diff
@@ -0,0 +1,109 @@
+""" Interface for generating an object describing geometry of mass spec acquisition process.
+"""
+
+def geometry_section(section_name):
+    """ Class decorator adding static property @section_name with the input value to a decorated class
+    """
+    def modifier(cls):
+        cls.section_name = section_name
+        return cls
+    return modifier
+
+
+class ACQ_GEOMETRY_KEYS(object):
```

Comment:
@iprotsyuk just as a comment, names tuples can also be used for grouping constants
https://stackoverflow.com/questions/29792189/grouping-constants-in-python#29794517

Reply:
yep, I'm aware of these. The use of `class` looks better to me in this particular case. I'd keep it this way, unless there're any other objections